node/src/actors/session/mod.rs

```diff
@@ -86,19 +86,22 @@ pub struct Session {
 
     /// Timestamp for requested blocks
     blocks_timestamp: i64,
+
+    /// Handshake maximum timestamp difference
+    handshake_max_ts_diff: i64,
 }
 
 /// Session helper methods
 impl Session {
     /// Method to create a new session
     pub fn new(
-        server_addr: SocketAddr,
-        remote_addr: SocketAddr,
+        (server_addr, remote_addr): (SocketAddr, SocketAddr),
```

Comment:
Hacks

Reply:
clippy things

Reply:
We can just disable that lint globally, I think that's better than doing these hacks all the time.